miprometheus/problems/video_to_class/seq_mnist_to_class/permuted_sequential_row_mnist.py

```diff
@@ -180,6 +180,20 @@ def collate_fn(self, batch):
     # get a sample
     sample = problem[0]
     print(repr(sample))
+
+    # test whether data structures match expected definitions
+    # images should be (batch size x sequence x channel x height x width)
+    # as this is a sample, we should have (sequence x channel x height x width) == (28, 1, 1, 28)
+    assert sample['images'].shape == torch.ones((28, 1, 1, 28)).shape, "Unit test failed! Expected images shape {} but got {}".format(torch.ones((28*28, 1, 1, 1)).shape, sample['images'].shape)
```

Comment:
Uuuu, nice! So we are really getting closer to something that is commonly understood as unit test ;)